src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -198,7 +199,11 @@ export default {
 		async getSecondCategoryData() {
 			this.secondCategoryLoans = await this.getMatchedLoans();
 			this.matchedLoansTotal = this.secondCategoryLoans.length;
-			if (this.matchedLoansTotal === 0) this.secondCategoryLoans = await this.getExpiringSoonAlmostFundedCombo(); // eslint-disable-line max-len
+			if (this.matchedLoansTotal < 3) {
+				const fallbackLoans = await this.getExpiringSoonAlmostFundedCombo();
+				this.secondCategoryLoans = [...this.secondCategoryLoans, ...fallbackLoans].slice(0, 9);
+			}
+			this.secondCategoryActive = true;
```

Comment:
I don't think this new flag is needed, since the source of the loans is still `secondCategoryLoans`

Reply:
yeah, but if the first assignment is less than 9, this is rendered with them and not updated later... other way is to have `secondCategoryLoans` as a computed prop and remove the flag

Reply:
Use a local variable in that method, then assign at the end.

Reply:
Good!